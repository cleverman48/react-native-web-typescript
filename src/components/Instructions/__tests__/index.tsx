import * as React from "react";
import { create } from "react-test-renderer";
import { Instructions } from "..";

describe("Instructions", () => {
  it("renders correctly with defaults", () => {
    const button = create(<Instructions instructions="123" />).toJSON();
    expect(button).toMatchSnapshot();
  });
});
