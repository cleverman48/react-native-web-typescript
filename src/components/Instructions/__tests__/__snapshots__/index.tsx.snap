// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions renders correctly with defaults 1`] = `
Array [
  <Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to React Native!
  </Text>,
  <Text
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  >
    To get started, edit ./src/containers/App/index.tsx
  </Text>,
  <Text
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }
  >
    123
  </Text>,
]
`;
